Breeze.UI/src/app/wallet/receive/receive.component.ts

```diff
@@ -14,13 +15,17 @@ import { NgbModal, NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
   styleUrls: ['./receive.component.css'],
 })
 
-export class ReceiveComponent {
-  constructor(private apiService: ApiService, private globalService: GlobalService, public activeModal: NgbActiveModal, private genericModalService: ModalService) {}
-
-  public address: any = "";
-  public copied: boolean = false;
+export class ReceiveComponent implements OnInit {
+  public address: any = '';
+  public copied = false;
```

Comment:
This is what I mentioned earlier, why get rid of the boolean type? Should probably have been named isCopied from the first place

Reply:
https://palantir.github.io/tslint/rules/no-inferrable-types/